### PR TITLE
docs: add Discord as an official communication channel

### PR DESCRIPTION
**Pull request by alex-okrushko**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

Adds Discord as a communication channel to the footer, README, CoC

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by alex-okrushko** [comment]
As discussed with @mgechev ✅

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 As contributors and maintainers of the Angular project, we pledge to respect everyone who contributes by posting issues, updating documentation, submitting pull requests, providing feedback in comments, and any other activities.
 
-Communication through any of Angular's channels (GitHub, Gitter, IRC, mailing lists, Google+, Twitter, etc.) must be constructive and never resort to personal attacks, trolling, public or private harassment, insults, or other unprofessional conduct.
+Communication through any of Angular's channels (GitHub, Discord, Gitter, IRC, mailing lists, Twitter, etc.) must be constructive and never resort to personal attacks, trolling, public or private harassment, insults, or other unprofessional conduct.
```
Comment (frederikprijck) [inline CODE_OF_CONDUCT.md:6]: As the link to Gitter is being removed from Conbtributing.md, should this also be removed from the list here ?

Of should the Gitter link be brought back ?

Feels like being inconsistent if not aligned.

Comment (mgechev) [reply CODE_OF_CONDUCT.md:6]: Gitter is still in the footer of the page. A few links have been replaced with Discord to encourage more folks to join the channel. 

Google+ has been discontinued for personal and brand accounts (2 April 2019; 18 months ago).

**Review by mgechev — APPROVED** [review]

**Review by aikithoughts — APPROVED** [review]
Reviewed-for: global-docs-approvers

**Review by IgorMinar — APPROVED** [review]
LGTM

**Pull request closed without merging**